community-expectations.md

```diff
@@ -0,0 +1,85 @@
+## Community Expectations
+
+Kubernetes is a community project and, as such, it is wholly dependent on
+its community to provide a productive, friendly and collaborative environment.
+
+The first and foremost goal of the Kubernetes community to develop orchestration
```

Comment:
This is one heck of a sentence. The first part is missing a verb - probably 'is' ?

I would break and start a new sentence after "distributed systems".  "...distributed systems.  A second, equally important goal..."

misspelled orchestration